shared/elements/gaia_lowstorage_dialog/script.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+/* global ComponentUtils, MozActivity */
+
+/**
+ * The gaia-confirm component displays a dialog in which the user has a
+ * choice to confirm or cancel the action. It may be displayed along with a
+ * title, description, and image. Buttons may also be configured.
+ * @requires GaiaButtons
+ */
+window.GaiaLowstorageDialog = (function(win) {
+  // Extend from the HTMLElement prototype
+  var proto = Object.create(HTMLElement.prototype);
```

Comment:
I didn't want to use the new style too much, I'm afraid of bad interactions between using the old gaia_confirm and new mechanisms.